web/src/logs.types.ts

```diff
@@ -49,12 +49,17 @@ export interface StreamsResult {
   result: Array<StreamLogData>;
 }
 
+type Chunk = Record<string, number>;
+type Store = Record<string, number> & { chunk?: Chunk };
+type Ingester = Record<string, number> & { store?: Store };
+type Querier = Record<string, number> & { store?: Store };
+
 export type QueryRangeResponse<T = MatrixResult | StreamsResult> = {
   status: string;
   data: T & {
     stats: {
-      ingester: Record<string, number>;
-      store: Record<string, number>;
+      ingester: Ingester;
+      querier: Querier;
       summary: Record<string, number>;
```

Comment:
I'm not sure if the API will always return these values, so setting this as optional will fallback the results to "NA" 

```suggestion
      ingester?: Ingester;
      querier?: Querier;
      summary?: Record<string, number>;
```